CONTRIBUTING.md

```diff
@@ -6,18 +6,33 @@ Learn the rules to follow when you contribute code or content to this project.
 
 ## Contributing process
 
-The contributing process in this project relies on the [GitHub flow](https://guides.github.com/introduction/flow/index.html). This means that you contribute through pull requests. When adding new code or content to this project, follow these rules:
+The contributing process in this project relies on the [GitHub flow](https://guides.github.com/introduction/flow/index.html). This means that you contribute through pull requests (PRs). When adding new code or content to this project, follow these rules:
 
 1. Fork this repository.
-2. Make your changes. Do not forgot about:
+1. Make your changes. Do not forgot about:
```

Comment:
`2. `

Reply:
such order is on purpose, thanks to it when you add/remove from list, you do not have to adjust, 
```
1.
1.
1.
```
renders to 

1.
2.
3.

Reply:
OK :(